src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -371,16 +371,22 @@ export default class AircraftInstanceModel {
     }
 
     /**
+     * Returns a true value if there is a match from the callsignToMatch
+     *
      * @for AircraftInstanceModel
      * @method matchCallsign
      * @param callsign {string}
      */
     matchCallsign(callsignToMatch) {
-        if (callsignToMatch === '*') {
+        const shouldMatchAnyCallsign = callsignToMatch === '*';
+
+        // checks to see if the given call sign matches the airline Id + flight number
+        if (shouldMatchAnyCallsign || _isEqual(callsignToMatch.toUpperCase(), this.callsign)) {
```

Comment:
_(already called out on slack, copying here for visibility)_
As long as we are comparing strings, I agree with Mr Quinn.  Lodash does a lot under the hood for [_isEqual()](https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L11545) that is overkill in this case.  Its a good thought here, but a simple strict equals (`===`) will definitely be more performant.